packages/dds/tree/src/simple-tree/simpleSchema.ts

```diff
@@ -26,6 +26,10 @@ export interface SimpleNodeSchemaBaseAlpha<
 > extends SimpleNodeSchemaBase<TNodeKind, TCustomMetadata> {
 	/**
 	 * Persisted metadata for this node schema.
+	 * @remarks
+	 * While this can be stored in the document, not all versions / configurations will do so.
+	 * Additionally this is not part of schema compatibility so collaborative session may see different persistedMetadata for the same node.
```

Comment:
Nit: might be worth expanding on "compatibility" here. Or linking to something that covers it in more detail.